### ISSUE TITLE
installation via pip doesn't install requirements

### ISSUE THREAD

**Issue by edgimar** [FabriceSalvaire/CodeReview#22]
Apparently there is a problem with how or where the requirements are currently specied -- when I run `pip3 install CodeReview`, none of the missing dependencies get installed (e.g. yaml, PyQt5, etc.).

**Comment by FabriceSalvaire** [comment]
Yes there is a typo

Todo: fix setup_data.py  `requires` -> `install_requires`

**Comment by kravemir** [comment]
I have the same issue. I've tested it on almost plain Ubuntu install, python3.6 packages are:

```
$ apt search python3.6 | grep installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

libpython3.6/yakkety,now 3.6.0~b2-1 amd64 [installed,automatic]
libpython3.6-dev/yakkety,now 3.6.0~b2-1 amd64 [installed,automatic]
libpython3.6-minimal/yakkety,now 3.6.0~b2-1 amd64 [installed,automatic]
libpython3.6-stdlib/yakkety,now 3.6.0~b2-1 amd64 [installed,automatic]
python3.6/yakkety,now 3.6.0~b2-1 amd64 [installed]
python3.6-dev/yakkety,now 3.6.0~b2-1 amd64 [installed]
python3.6-minimal/yakkety,now 3.6.0~b2-1 amd64 [installed,automatic]
python3.6-venv/yakkety,now 3.6.0~b2-1 amd64 [installed]
```

And, OS version:
```
$ cat /etc/os-release | grep NAME
NAME="Ubuntu"
PRETTY_NAME="Ubuntu 16.10"
VERSION_CODENAME=yakkety
UBUNTU_CODENAME=yakkety
```

I ran it in venv (`python3.6 -m venv __venv__`)

**Comment by FabriceSalvaire** [comment]
fixed in 28ff141d688167871d57a0a7a60241e422479043

uploaded new version
